Jenkinsfile

```diff
@@ -0,0 +1,252 @@
+
+pipeline {
+    agent any
+    options {timestamps()}
+    /*triggers {
+        pollSCM('H/2 * * * *')
+    } */
+    stages {
+        stage ('fetch-opa-psm2')  {
+             steps {
+                 withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) { 
+                     dir('opa-psm2-lib') {
+
+                        checkout changelog: false, poll: false, scm: [$class: 'GitSCM', \
+                        branches: [[name: '*/master']], \
+                        doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], \
+                        userRemoteConfigs: [[url: 'https://github.com/intel/opa-psm2.git']]]                        
```

Comment:
Why does psm need special handling, but verbs does not?  How is verbs installed?

Reply:
The nodes that we use to test already have verbs dependencies installed from the distro. There was some issue with psm2 and distro libpsm2 libraries and we decided to use upstream opa-psm2 instead which didn't have the problem.

Reply:
That makes sense.  This file is being placed at the root of the repo.  Since it is dependent on the Intel systems, can it move under contrib/intel?  Or does Jenkins not like that?

Reply:
It can be placed anywhere in the repo. contrib/cray serves the same purpose.